scissors/main.c

```diff
@@ -0,0 +1,86 @@
+#include <stdio.h>
+#include <time.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+enum {
+	NONE	= -1,
+	ROCK	= 0,
+	SCISSORS= 1,
+	PAPER	= 2,
+	OPTS_MAX
+}options;
+
+const int rules_martix[OPTS_MAX][OPTS_MAX] =
+{
+	{NONE,	ROCK, PAPER},
```

Comment:
If you add an another enum `winner` (for example, DRAW, FIRST, SECOND) and fill the matrix appropriately, you can use `switch (rules_martix[pc_choise][human_choise])` instead of if - else if chain,

Reply:
Also if fill matrix with winer message (like "_You win. smth beats smth_"), we can do without any confitional constructs)

Reply:
Left as is.

Reply:
Let it be